xcdat/regridder/grid.py

```diff
@@ -493,6 +495,7 @@ def create_grid(
             lon_bnds = lon_bnds.copy()
 
         data_vars["lon_bnds"] = lon_bnds
+        lon.attrs["bounds"] = lon_bnds.name
 
     grid = xr.Dataset(data_vars=data_vars, coords={"lat": lat, "lon": lon})
```

Comment:
The `"bounds"` attr for a coord var must be set to the name of the bounds data var. Otherwise, the subsequent call to `grid.bounds.add_missing_bounds()` will attempt to add bounds even if they already exist because it thinks the bounds don't exist.

If this `"bounds"` attribute is _not_ set, it breaks the test below.

https://github.com/xCDAT/xcdat/blob/e366c9ac6bb1631c251b64d00e733ae0a3919d61/tests/test_regrid.py#L533-L543

Reasons why:

* Bounds already exist but `add_missing_bounds()` doesn't see it because the `"bounds"` attr is not set
* `_add_bounds()` breaks  because there is only 1 coordinate point for `lat` and `lon` (conditional below)

What it should be doing instead:
* Ignore `add_missing_bounds()` altogether

https://github.com/xCDAT/xcdat/blob/e366c9ac6bb1631c251b64d00e733ae0a3919d61/xcdat/bounds.py#L266-L270